docs/guides/http2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Leveraging HTTP/2 with tailcall
+---
+
+HTTP/2 is a modern protocol that significantly improves web performance by introducing features like multiplexing, header compression, and more efficient connection handling. Tailcall, through its @http operator, enables seamless integration and utilization of HTTP/2 capabilities for both inbound and outbound API requests.
+
+## Certificate and Key Generation
```

Comment:
I prefer a separate guide to setup Tailcall over HTTPS. Also HTTPS should be optional after this :

https://github.com/tailcallhq/tailcall/pull/842